rfd/0153-resource-guidelines.md

```diff
@@ -477,6 +478,55 @@ func (s *FooService) ListFoos(ctx context.Context, pageSize int, pageToken strin
 }
 ```
 
+### Event stream mechanism
```

Comment:
I'm a little split because whilst it's certainly a necessary pre-requisite for Cache support, it's not necessarily only beneficial for the Cache, e.g you may have other reasons that clients need to be able to watch your resource.

Reply:
Ah - that does sound likely - I wasn't aware of this. I'll move it under the cache heading.